bridge/opencensus/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry/OpenCensus Bridge
+
+[![PkgGoDev](https://pkg.go.dev/badge/go.opentelemetry.io/otel/bridge/opencensus)](https://pkg.go.dev/go.opentelemetry.io/otel/bridge/opencensus)
+[Example](https://github.com/open-telemetry/opentelemetry-go/blob/main/example/opencensus/main.go)
```

Comment:
I suggest removing the [`example/opencensus`](https://github.com/open-telemetry/opentelemetry-go/blob/main/example/opencensus/main.go) and create additional test examples (or refine existing ones) instead.

Reply:
I'm not opposed to more test examples, but I feel like the one in `example/opencensus` is a bit different.
It does a lot of setup, and is basically runnable as-is, when test examples are meant to be lighter, and to show one specific thing only.

So it seems to me both have their use.